packages/locales/lib/human/de.json

```diff
@@ -585,11 +585,12 @@
   "delete": "löschen",
   "name": "Name",
   "only_confirmed": "nur bestätigte",
-  "only_exclude_grunts": "Grunzer ausschließen",
-  "only_exclude_leaders": "EAnführer ausschließen",
+  "only_exclude_grunts": "Rüpel ausschließen",
+  "only_exclude_leaders": "Bosse ausschließen",
   "rocket_pokemon": "Crypto-Pokemon",
   "decoy": "Täusch-Rüpel",
   "s2_cell_limit_0": "Sie haben versucht, mehr als 20.000 Zellen zu generieren ({{variable_0}})",
+  "s2_cell_zoom_limit": "Einige Zellen sind zu klein, um in dieser Zoomstufe angezeigt zu werden.",
```

Comment:
Zoomlevel? Or do we use Stufe in several places?

Reply:
don't think so... changed